src/Workspaces/Core/Portable/FindSymbols/Shared/AbstractSyntaxIndex_Persistence.cs

```diff
@@ -19,7 +18,7 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     internal partial class AbstractSyntaxIndex<TIndex> : IObjectWritable
     {
         private static readonly string s_persistenceName = typeof(TIndex).Name;
-        private static readonly Checksum s_serializationFormatChecksum = Checksum.Create("27");
+        private static readonly Checksum s_serializationFormatChecksum = Checksum.Create("28");
```

Comment:
Guessing this, and the next 4 changes aren't supposed to be here :P

Reply:
no.  this is relevant.  we're changing what we store in the index.  so we need this value to cahnge so that all the indices on everyones' machines are updated :)

Reply:
🤯